src/Files.Core/Services/Settings/IAppearanceSettingsService.cs

```diff
@@ -24,6 +24,11 @@ public interface IAppearanceSettingsService : IBaseSettingsService, INotifyPrope
 		/// </summary>
 		bool UseCompactStyles { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value for the app theme mode.
+		/// </summary>
+		String AppThemeMode { get; set; }
```

Comment:
```suggestion
		String AppTheme { get; set; }
```
I think we can remove the "Mode" part.